sites/all/themes/wille/templates/system/html.tpl.php

```diff
@@ -59,5 +59,6 @@
   <?php print $page; ?>
   <?php print $scripts; ?>
   <?php print $page_bottom; ?>
+  <script src="<?php print drupal_get_path('theme', 'wille') . '/overlay/overlay.js?2'; ?>"></script>
```

Comment:
Do they really want this on all pages? If not, we have to find a different way to insert the `script` element.

Reply:
They would like to test it only on the frontpage. Do you have any suggestions on how to do that?

Reply:
@yepzdk, https://www.drupal.org/forum/support/module-development-and-code-questions/2011-10-01/adding-javascript-to-front-page-only#comment-5064274 may be the way to do it. Or at least worth a try.

Reply:
Thanks @rimi-itk